source/numerics.tex

```diff
@@ -8025,7 +8025,7 @@
 template, which is one-dimensional.
 The set of one-dimensional index values specified by a
 \tcode{gslice}
-are $$k = s + \sum_ji_jd_j$$
+are \[k = s + \sum_ji_jd_j\]
```

Comment:
While you're here, make this a bit easier to read:

    \[ k = s + \sum_j i_j d_j \]

Also add spacing after `\[` below.